common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -31,19 +31,28 @@ class ProjectCard extends React.PureComponent<Props> {
     );
   }
 
+  _goToDetailsView(): void {
+    window.location.replace('/project/' + this.props.project.id);
```

Comment:
Can we inject a regular link onto the cards?  Usability-wise, people will want to do things like ctrl-click, and see the hand when they hover over something that's a link.

Reply:
Makes sense! will fix